src/main/resources/assets/skyblocker/lang/en_us.json

```diff
@@ -168,6 +168,8 @@
   "text.autoconfig.skyblocker.option.locations.spidersDen.relics": "Hidden Relics Helper",
   "text.autoconfig.skyblocker.option.locations.spidersDen.relics.enableRelicsHelper": "Enable Hidden Relics Helper",
   "text.autoconfig.skyblocker.option.locations.spidersDen.relics.highlightFoundRelics": "Highlight found relics",
+  "text.autoconfig.skyblocker.option.locations.garden": "Garden",
+  "text.autoconfig.skyblocker.option.locations.garden.dicerTitlePrevent": "Enable dicer title prevent",
```

Comment:
Can you capitalize all the words in the string and while you're at it also the found relics string?